app/controllers/podcasts_controller.rb

```diff
@@ -18,6 +18,18 @@ def create
     end
   end
 
+  def approve_podcast
+    podcast = Podcast.find(params[:id])
+    podcast.update(active: true) 
+    if podcast.save
+      redirect_to '/dashboard'
+      flash[:success] = 'Application approved'
+    else
```

Comment:
Sad path but cant really think of an instance where a podcast wouldnt be approved or atleast for MVP we dont have that.